js/SubmitAllView.js

```diff
@@ -22,10 +22,10 @@ export default class SubmitAllView extends Backbone.View {
 
   render() {
     this.articleView.$el.addClass('no-submit-buttons');
-    const {
-      buttonText,
-      ariaLabel
-    } = Adapt.course.get('_buttons')._submit;
+    const buttons = this.model.get('_button');
+    const courseSubmitButtons = Adapt.course.get('_buttons')._submit;
+    const buttonText = buttons.buttonText || courseSubmitButtons.buttonText;
+    const ariaLabel = buttons.ariaLabel || courseSubmitButtons.ariaLabel;
```

Comment:
Add [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) for when the buttons object doesn't exist:
```suggestion
    const buttonText = buttons?.buttonText || courseSubmitButtons.buttonText;
    const ariaLabel = buttons?.ariaLabel || courseSubmitButtons.ariaLabel;
```

Reply:
Otherwise, 👍